docs/claim-transparency.md

```diff
@@ -0,0 +1,345 @@
+which is meaningless if the format were to be used for specifying a security or privacy claim.
+Builder and buildConfig are other fields that are irrelevant to security or privacy claims.
+Similarly the field names in the schema suggested for claims are meaningless in the context of a
+provenance statement.
+
+| Field in a Claim statement | Field in a SLSA provenance | Comments |
+|:----------------|:---------------|:-----------------------------------------------------------|
+| issuer | builder | SLSA distinguishes between the builder and signer. In the context of claims, we expect the issuer and the signer to be the same entity.|
+| claimType | buildType | Both define the meanings of the other fields in the predicate.|
+| claimSpec | buildConfig | Both provide a flexible way of supporting different types of content (claims, and build processes).|
+| evidence | materials | Optional list of (a subset of ) additional artifacts that influenced the statement. |
+
+## Comparison to RATS
+
+The Remote ATtestation procedureS (RATS) working group has provided an [architecture](https://datatracker.ietf.org/doc/html/draft-ietf-rats-architecture)
+and glossary of concepts related to remote attestation. [This cheatsheet](https://github.com/thomas-fossati/rats-cheatsheet)
+and [this slides deck](https://confidentialcomputing.io/wp-content/uploads/sites/85/2021/09/IETF-Remote-Attestation-Architecture-Overview.pdf)
+give an overview of the architecture and the main concepts. RATS has many concepts similar
+to the ones in our design, but seems to be focused on claims and evidence that are generated and
+consumed automatically. Claims and evidence in RATS are designed to be used for remote attestation.
+The claims in our binary transparency ecosystem, however, are not limited to the ones used for
+remote attestation. We target a wider range of use cases.
+
+## Examples
+
+### Endorsement Claims
+
+With the schema given for claims, an endorsement statement can be seen as a claim where the
+claimSpec is empty. In this case, the claimType has to clearly state that the claim is an
+endorsement statement. An endorsement statement, in the evidence field, may provide links to a
+provenance statement. In this case, the provenance statement must have the same subject as the one
+in the endorsement statement. If the linked provenance is signed, the evidence list may as well
+include a reference to a Rekor log entry corresponding to the provenance.
+
+```json
+{
+  "_type": "https://in-toto.io/Statement/v0.1",
+  "subject": [
+    {
+      "name": "oak_functions-012a5206e5ab35d2778832638519441dd27664da",
+      "digest": {
+        "sha256": "01b792106ef1f61eece3a666ac6069875fc90b942fefc3fe931f016395bb6c88"
+      }
+    }
+  ],
+
+  "predicateType": "https://github.com/project-oak/transparent-release/schema/claim/v1",
+  "predicate": {
+    "issuer": {
+      "id": "mailto:oak-team@google.com"
+    },
```

Comment:
Does this need to match something about the key that is ultimately used to sign the statement? I am quite worried about the fact that anyone can put whatever they want here, and even though in principle the signature is the only thing that matters, validators will mistakenly assume that this is an authoritative representation of the issuer. (e.g. this caused a lot of [issues](https://www.websecuritylens.org/common-jwt-implementation-mistakes-and-how-to-exploit-them/#:~:text=Not%20Validate%20the,as%20an%20invalid.) with JWT in the past)

Reply:
Yes. I just removed a sentence related to this point. This is a design choice that we have to clarify separately from this documentation PR. Feel free to create an issue for it.